tests/unit_tests/dagmc/test_model.py

```diff
@@ -0,0 +1,116 @@
+import pkg_resources
+from pathlib import Path
+
+import numpy as np
+import pytest
+
+import openmc
+from openmc import ZPlane, YPlane, XPlane, Cell
+
+pytestmark = pytest.mark.skipif(
+    not openmc.lib._dagmc_enabled(),
+    reason="DAGMC CAD geometry is not enabled.")
+
+
+def set_dagmc_model():
```

Comment:
Let's turn this into a `pytest.fixture` that can be passed to the test functions below.